deployments/chef/kitchen.yml

```diff
@@ -175,6 +175,8 @@ suites:
         auto_instrumentation_enable_profiler_memory: true
         auto_instrumentation_enable_metrics: true
         auto_instrumentation_otlp_endpoint: http://0.0.0.0:4317
+        auto_instrumentation_otlp_endpoint_protocol: grpc
```

Comment:
Don't we want to use `http://0.0.0.0:4318` with `http/protobuf` everywhere by default now?

Reply:
> Don't we want to use `http://0.0.0.0:4318` with `http/protobuf` everywhere by default now?

This is a test config just to check if the env vars are configured correctly.